sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -95,7 +88,7 @@ search_metrics AS (
     SUM(search_count) AS search_count,
     SUM(search_with_ads) AS search_with_ads,
   FROM
-    unioned
+    clients_last_seen_unioned AS unioned
```

Comment:
I had introduced the `clients_last_seen_unioned` name in my recent change. Now that we're removing the extra step, I think it would make sense to name `clients_last_seen_unioned` back to just `unioned` at the start of the query and avoid having to alias here.

Reply:
Sounds good. I initially left the `clients_last_seen_unioned` name out of clarity of the contents, but it can be easily read from the query.